src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -625,6 +625,29 @@ public void failAllDeletedOnes() throws Exception {
         d.assertItemNames(2, "A");
     }
 
+    @Issue("JENKINS-73930")
+    @Test
+    public void disabledWarningFromUiViews() throws Exception {
+        LockedDownSampleComputedFolder folder = r.jenkins.createProject(LockedDownSampleComputedFolder.class, "d");
+        assertFalse("by default, a folder is disabled", folder.isDisabled());
+        folder.getViews().forEach(view -> {
+            try {
+                assertNull(r.createWebClient().goTo(view.getViewUrl()).getElementById("disabled-message"));
+            } catch (Exception e) {
+                Assert.fail();
```

Comment:
Better to switch to a plain `for` loop so you do not need to worry about checked exceptions.